h/routes.py

```diff
@@ -40,6 +40,7 @@ def includeme(config):
     config.add_route('admin_cohorts_edit', '/admin/features/cohorts/{id}')
     config.add_route('admin_groups', '/admin/groups')
     config.add_route('admin_groups_create', '/admin/groups/new')
+    config.add_route('admin_groups_edit', '/admin/groups/edit/{id}')
```

Comment:
Anticipating that we'll want an admin-edit-group functionality in the admin area in the future, I have generalized the route and controller, roughly following the (most recent, I think) OAuth admin pattern.